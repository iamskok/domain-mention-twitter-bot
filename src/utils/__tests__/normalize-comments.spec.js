import normalizeComment from '../normalize-comment';
import tweet from '../../tests/options/firebaseTweet';

describe('normalizeComment', () => {
  it('returns comment object', () => {
    expect(normalizeComment(tweet.comment, [])).toMatchSnapshot();
  });
});
