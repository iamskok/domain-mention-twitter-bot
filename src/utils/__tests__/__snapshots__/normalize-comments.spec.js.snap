// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`normalizeComment returns comment object 1`] = `
Object {
  "avatar": null,
  "content": Object {
    "text": null,
  },
  "handle": "@netlify",
  "name": null,
  "profile": "https://twitter.com/netlify",
  "timestamp": null,
  "url": "https://twitter.com/netlify/statuses/1324842247691657216",
}
`;
